src/components/RadioButton/Bounceable.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {PropsWithChildren} from 'react'
+import {ViewStyle, StyleProp, Pressable, PressableProps, View} from 'react-native'
+import Animated, {withSpring, useSharedValue, useAnimatedStyle} from 'react-native-reanimated'
+
+const AnimatedPressable = Animated.createAnimatedComponent(Pressable)
+
+type CustomStyleProp = StyleProp<ViewStyle> | Array<StyleProp<ViewStyle>>
+
+export interface IBounceableProps extends Omit<PressableProps, 'children'>, PropsWithChildren {
```

Comment:
I remember the `PressableProps` includes `children` but why do we omit it and `extends PropsWithChildren`

Reply:
@anhle10051996 could you answer?

Reply:
I guess, these are different the children type

Reply:
` children?: ReactNode | undefined`
 and
```
 children?:
    | React.ReactNode
    | ((state: PressableStateCallbackType) => React.ReactNode)
    | undefined;
```

Reply:
Hi @ThinhKimVo, can you change the code to just extends PressableProps

```suggestion
export interface IBounceableProps extends PressableProps {
```